app/policies/liquid_tag_policy.rb

```diff
@@ -0,0 +1,30 @@
+# Intentionally not inheriting from ApplicationPolicy because liquid tags behave
+# differently than the typical Model/Controller dynamic that Pundit assumes.
+class LiquidTagPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def initialize?
+    return true unless record.class.const_defined?("VALID_ROLES")
+    raise Pundit::NotAuthorizedError, "No user found" unless user
+    # Manually raise error to use a custom error message
+    raise Pundit::NotAuthorizedError, "User is not permitted to use this liquid tag" unless user_permitted_to_use_liquid_tag?
+
+    true
```

Comment:
Why do we need to return `true` here? Wouldn't `user_permitted_to_use_liquid_tag?` return `true` as the last thing that is invoked here? (Unless of course it was `false` and then `raise`d the `Pundit::NotAuthorizedError`?

Reply:
It returns `nil` because it's a conditional statement.

Reply:
Ah, of course. 🤦‍♀️